AzureADSample/AzureADSample/View/SettingsPage.xaml

```diff
@@ -0,0 +1,31 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<ContentPage xmlns="http://schemas.microsoft.com/dotnet/2021/maui"
+             xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml"
+             xmlns:viewmodel="clr-namespace:AzureADSample.ViewModel"
+             x:Class="AzureADSample.Views.SettingsPage"
+             x:DataType="viewmodel:SettingsPageViewModel"
+             Title="SettingsPage">
```

Comment:
Let's change the name of the SettingsPage Title and file names to something that's more representative. For instance, "Profile" and "ProfilePage" 
```suggestion
             x:Class="AzureADSample.Views.ProfilePage"
             x:DataType="viewmodel:ProfilePageViewModel"
             Title="ProfilePage">
```

of course, be sure to update the actual file names and all their references as well

Reply:
Got it!

Reply:
nitpick: Add a space between Profile and Page for the Title - my bad for not capturing that earlier! For what's presented on the UI, we want to ensure all text is as readable as possible

`Title="Profile Page"`